### DISCUSSION TITLE
Example sentence for 金輪際 has a weird English translation

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#67] [Errors and Corrections]
JP: 彼は金輪際君を助けてくれないだろう。

EN: You can make a book on it that he won't help you.

---

Looks like "you can make a book on it" might be an expression in English but honestly I am not familiar with it and the few results I find on Google don't inspire much confidence (also the usage seems to be a bit different too). Overall, I think the sentence is just unnecessarily weird and confusing. It would be nice to use a different sentence instead. I looked at tatoeba and [this one](https://tatoeba.org/en/sentences/show/893234) seems to be more straightforward (including the English translation).

**Comment by stephenmk** [comment]
I removed the priority flag for 金輪際 from that sentence (#[108143](https://tatoeba.org/en/sentences/show/108143)) and added a new index string for sentence #[893234](https://tatoeba.org/en/sentences/show/893234)

> 私(わたし) は(#2028920)[01] 金輪際[01]~ カラオケバー では(#1008450)[03] 歌う[01]{歌わない}

CC @jimmubreen

**Comment by JMdictProject** [comment]
Thanks.
